docs/spec/qualifiers.rst

```diff
@@ -213,35 +213,44 @@ following should be allowed::
 Syntax
 ^^^^^^
 
-``Annotated`` is parameterized with a type and an arbitrary list of
-Python values that represent the annotations. Here are the specific
-details of the syntax:
+``Annotated`` is parameterized with a *base type* and an arbitrary list of
+Python values that represent *metadata* for the type::
```

Comment:
The phrase "for the type" is problematic here. It implies that the metadata must apply to the *type*. That hasn't previously been specified, and some existing use cases for `Annotated` treat the metadata as applying to the *symbol*, not the *type*. This phrasing therefore constrains the use cases of `Annotated`. Is this your intent? If so, I think this deserves a much broader discussion before we make such a change.

Reply:
```suggestion
Python values that represent *metadata*::
```
I definitely don't intend to narrow the possible use cases for Annotated. Would simply removing "for the type" work?

Reply:
Yes, I think that works.

Reply:
You could also consider something like

```suggestion
Python values that represent *associated metadata*::
```

which would leave open the possibility that it could be the _type_ that the metadata is associated with, but it could also be the _symbol_ that the metadata is associated with

Reply:
Thanks, I added "associated" (but outside the italics because the term being introduced is "metadata", not "associated metadata").